alvr/client_openxr/src/lib.rs

```diff
@@ -353,6 +353,10 @@ pub fn entry_point() {
 
                         lobby.update_reference_space();
 
+                        let relative_head_pose = interaction_context.read().relative_head_pose;
+                        interaction_context.write().relative_head_pose =
```

Comment:
You can keep the `*=` operator in theory

Reply:
> error[E0368]: binary assignment operation `*=` cannot be applied to type `alvr_common::Pose`